examples/type-conversion/type_conversion.bal

```diff
@@ -1,71 +1,106 @@
 import ballerina/io;
 
 type Person record {
-    string name = "";
-    int age = 0;
+    string name;
+    int age;
 };
 
 type Employee record {
-    string name = "";
-    int age = 0;
-    int empNo = 0;
+    string name;
+    int age;
+    int empNo;
 };
 
-function convertType(Employee emp) returns () {
-    // The `convert()` creates a new value and changes its type without editing provided value's inherent type.
-    Person|error empPerson = Person.convert(emp);
-    io:println("empPerson name: ",
-                (empPerson is Person) ? empPerson["name"] : empPerson.reason());
+// Function to attempt converting an `anydata` record `Employee`, to an `anydata` record `Person`.
```

Comment:
What does `anydata` record mean?

Reply:
A record that is `anydata`. Do we have a better term?

Reply:
Does it matter whether we say `anydata` or not? Cant we simply say -

```
// Function to attempt converting an `Employee`, to a `Person`.
```

Reply:
Wanted to emphasize that it is only allowed with `anydata`.

Reply:
`anydata` compatible?

Reply:
Not sure if it implies the same? We are talking about the type right, not compatibility?